cmd/target-browser/cmd/resource-metadata.go

```diff
@@ -0,0 +1,79 @@
+package cmd
+
+import (
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+
+	targetBrowser "github.com/trilioData/tvk-plugins/tools/target-browser"
+)
+
+func init() {
+	getCmd.AddCommand(resourceMetadataCmd())
+}
+
+// resourceMetadataCmd represents the resource-metadata command
+func resourceMetadataCmd() *cobra.Command {
+	var cmd = &cobra.Command{
+		Use:   ResourceMetadataCmdName,
+		Short: "Get Backup high level metadata details",
+		Long:  `Performs GET operation on target-browser's '/resource-metadata' API and gets Application YAMLs from backup in JSON format`,
+		Example: `  # Get resource-metadata details of specific backup
+  kubectl tvk-target-browser get resource-metadata --backup-uid <uid> --backup-plan-uid <uid> --name <name> --group <group> 
+	--version <version> --kind <kind>
+`,
+		RunE: getResourceMetadata,
+	}
+
+	cmd.Flags().StringVar(&group, groupFlag, groupDefault, groupUsage)
+
+	cmd.Flags().StringVar(&backupPlanUID, BackupPlanUIDFlag, backupPlanUIDDefault, backupPlanUIDUsage)
+	err := cmd.MarkFlagRequired(BackupPlanUIDFlag)
+	if err != nil {
+		log.Fatalf("Invalid option or missing required flag %s - %s", BackupPlanUIDFlag, err.Error())
```

Comment:
no need to mark this flag required.. use default value empty string.

Reply:
Kept it until server side changes are done